data/mainnet/tokens.js

```diff
@@ -3108,7 +3108,8 @@ module.exports = {
   mirrorAAPL: {
     chain: CHAINS_ID.ETH_MAINNET,
     isDegen: true,
-    category: VAULT_CATEGORIES_IDS.MSTONKS,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
+    inactive: true,
     logoUrl: './icons/mirror-aapl.png',
     apyIconUrls: ['./icons/mir.png'],
```

Comment:
Forgot to mention this before, but please also empty both these apy fields to just empty lists: `[],`, please also do it for the vaults you moved to inactive before. Right now the UI still shows the symbols for the rewards: 
![image](https://user-images.githubusercontent.com/34768229/136648666-9e125935-704f-4e42-9de9-049e78ceab8f.png)